components/atoms/ButtonAtoms/index.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function Buttons(props) {
+  const LargeButton = styled.button`
+    background-color: ${props => props.background || '#CE5374'};
```

Comment:
Considering props is already defined in the outer Buttons function, re-declaring props here will be repetition. 
One way to go about it is to remove the props from line 4